lib/components/form/advanced-settings-panel.tsx

```diff
@@ -0,0 +1,288 @@
+
+  .date-time-modal {
+    padding: 0;
+
+    .main-panel {
+      margin: 0;
+
+      button {
+        padding: 6px 0;
+      }
+
+      .date-time-selector {
+        margin: 15px 0;
+      }
+    }
+  }
+`
+
+const AdvancedSettingsPanel = ({
+  closeAdvancedSettings,
+  currentQuery,
+  enabledModeButtons,
+  innerRef,
+  modeButtonOptions,
+  modeSettingDefinitions,
+  modeSettingValues,
+  onPlanTripClick,
+  routingQuery,
+  setQueryParam
+}: {
+  closeAdvancedSettings: () => void
+  currentQuery: any
+  enabledModeButtons: string[]
+  innerRef: RefObject<HTMLDivElement>
+  modeButtonOptions: ModeButtonDefinition[]
+  modeSettingDefinitions: ModeSetting[]
+  modeSettingValues: ModeSettingValues
+  onPlanTripClick: () => void
+  routingQuery: () => void
+  setQueryParam: (evt: any) => void
+}): JSX.Element => {
+  const baseColor = getBaseColor()
+  const accentColor = baseColor || blue[900]
+
+  const intl = useIntl()
+  const closeButtonText = intl.formatMessage({
+    id: 'components.BatchSearchScreen.closeAdvancedPreferences'
+  })
+  const headerText = intl.formatMessage({
+    id: 'components.BatchSearchScreen.advancedHeader'
+  })
+
+  // @ts-expect-error Context not typed
+  const { ModeIcon } = useContext(ComponentContext)
+
+  const planTrip = () => {
+    alertUserTripPlan(intl, currentQuery, onPlanTripClick, routingQuery)
+    closeAdvancedSettings()
+  }
+
+  const closeAndAnnounce = () => {
+    closeAdvancedSettings()
+    toast.success(intl.formatMessage({ id: 'actions.user.preferencesSaved' }))
+  }
+
+  const processSettings = (settings: ModeSetting[]) =>
+    settings.map(
+      pipe(
+        populateSettingWithIcon(ModeIcon),
+        populateSettingWithValue(modeSettingValues),
+        addCustomSettingLabels(intl)
+      )
+    )
+
+  const globalSettings = modeSettingDefinitions.filter((x) => !x.applicableMode)
+  const processedGlobalSettings = processSettings(globalSettings)
+
+  const globalSettingsComponents = processedGlobalSettings.map(
+    (setting: ModeSetting) => (
+      <ModeSettingRenderer
+        key={setting.key}
+        onChange={onSettingsUpdate(setQueryParam)}
+        setting={setting}
+      />
+    )
+  )
+
+  const processedModeSettings = processSettings(modeSettingDefinitions)
+  const processedModeButtons = modeButtonOptions.map(
+    pipe(
+      addModeButtonIcon(ModeIcon),
+      addSettingsToButton(processedModeSettings),
+      setModeButtonEnabled(enabledModeButtons)
+    )
+  )
+
+  return (
+    <PanelOverlay className="advanced-settings" ref={innerRef}>
+      <HeaderContainer>
+        <PageTitle title={headerText} />
+        <h1 className="header-text">{headerText}</h1>
+        <CloseButton
+          aria-label={closeButtonText}
+          onClick={closeAndAnnounce}
+          title={closeButtonText}
+        >
+          <Close size={22} />
+        </CloseButton>
+      </HeaderContainer>
+      <DtSelectorContainer>
+        <DateTimeModal />
```

Comment:
Is it possible to remove the blue background? Very confusing for the date time selector to be separated from other content by background color when all other seconds are demarcated by header text

Reply:
Just to clarify, the header text in most deployments will not be visible to sighted users. The header text you're seeing is a custom configuration. It will be hidden by default. 

The blue I'm happy to nix if that's what folks want, although I think it looks nice and makes the DT modal feel like more of its own tool rather than three different buttons? I added it because the upcoming design utilizes the blue background and this felt like a way to bridge the gap. Curious what other folks think.

Reply:
Internal consistency https://www.nngroup.com/articles/ten-usability-heuristics/ #8, #6, #4